generic/tests/netperf.py

```diff
@@ -222,10 +252,13 @@ def _pin_vm_threads(vm, node):
             password = params_tmp["password"]
             username = params_tmp["username"]
             env_setup(i, ip_dict[i], username, shell_port, password)
+    mtu = int(params.get("mtu", "1500"))
+    if mtu != 1500:
+        mtu_set(mtu)
```

Comment:
This is incorrect, the actual value may not be the same as you set.

Reply:
I am not sure what's you mean, here only mtu is not 1500 then will can call mtu_set func.

Reply:
I mean the actual mtu value of the nic on your test system may not be 1500 (although you set `mtu` to 1500). You should set it by hand in this situation.

Reply:
Even without there lines, the actually mtu value may not be 1500,it depends on a fresh host.

Reply:
That's why you must set the actual value to what you specified.

Reply:
the actual value could be false (changed by some tests), anyway, 1500 should be default value which is need to be tested by default, so I change the logic, call mtu_set (1500) to avoid the mistake value. please help a check.